src/box/memtx_hash.cc

```diff
@@ -171,8 +171,9 @@ hash_iterator_eq(struct iterator *it, struct tuple **ret)
 		return 0;
 	struct txn *txn = in_txn();
 	struct space *sp = space_by_id(it->space_id);
-/********MVCC TRANSACTION MANAGER STORY GARBAGE COLLECTION BOUND START*********/
 	*ret = memtx_tx_tuple_clarify(txn, sp, *ret, it->index, 0);
+/********MVCC TRANSACTION MANAGER STORY GARBAGE COLLECTION BOUND START*********/
+	memtx_tx_story_gc();
```

Comment:
We should do the same for `WRAP_ITERATOR_METHOD` macro.

Reply:
Thanks, missed that one: added garbage collection to `WRAP_ITERATOR_METHOD`.